dist/s_void

```diff
@@ -2,8 +2,7 @@
 
 t=__wt.$$
 trap 'rm -f $t' 0 1 2 3 13 15
-
-cd ..
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"/.. || exit $?
```

Comment:
What about using the new `grep` here?

Reply:
1. The new grep greatly improves performance when matching multiple (especially, many!) patterns. On a single pattern it works about the same.
2. The run time of `s_vois` is just 5 seconds, it wouldn't improve the overall run time.